README.md

```diff
@@ -126,17 +125,33 @@ Then create a new env with `python3 -m venv <name of your venv>`
 
 Activate the venv with `source <name of your venv>/bin/activate`
 
+Install poetry with `pip install poetry`
```

Comment:
Since it's [recommended](https://python-poetry.org/docs/) to keep Poetry in a separate virtual environment that is not the project environment, maybe we should advise to use `pipx` to install Poetry? Not sure if there's another, more convenient way.